installer/get.sh

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env bash
+
+# FlowFuse Device Agent Installer Script
+# This script detects the OS and architecture, then downloads the appropriate binary
+# from the FlowFuse Device Agent releases page on GitHub.
+
+set -e
+
+# GitHub repository information
+REPO_OWNER="FlowFuse"
+REPO_NAME="device-agent"
+RELEASE="1.0.0"
+RELEASE_TAG="installer-v1.0.0"
```

Comment:
So the plan will be to bump this value as part of the release?

If so can the `RELEASE_TAG` value be built by including `RELEASE` e.g.

```suggestion
RELEASE="1.0.0"
RELEASE_TAG="installer-v$RELEASE"
```

Then we only need to update one line